### PR TITLE
Hide Keystore Release password on Android's Export window

### LINKED ISSUE #8813: Hide Keystore password on android export window
### Describe the project you are working on

Android game

### Describe the problem or limitation you are having in your project

Release key password shown in editor

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Just hide password behind ***

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Now it's look like this
image
I want to be like this
image

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

Because it's security

### PR DESCRIPTION
**Pull request by Mickeon**
Resolves https://github.com/godotengine/godot-proposals/issues/8813

### CODE REVIEW

File: platform/android/export/export_plugin.cpp [hunk 1]
```diff
@@ -1826,7 +1826,7 @@ void EditorExportPlatformAndroid::get_export_options(List<ExportOption> *r_optio
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/debug_password", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/release", PROPERTY_HINT_GLOBAL_FILE, "*.keystore,*.jks", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET), ""));
 	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/release_user", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET), ""));
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/release_password", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "keystore/release_password", PROPERTY_HINT_PASSWORD, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_SECRET), ""));
```
Comment (m4gr3d) [inline platform/android/export/export_plugin.cpp:1829]: The change makes sense! 

For consistency, can you update the export debug password and the debug password in the editor settings to match.

**Review by m4gr3d — CHANGES_REQUESTED** [review]

**Comment by Mickeon** [comment]
Updated the PR with the above requests

**Comment by Mickeon** [comment]
Updated the PR with the above requests

**Review by m4gr3d — APPROVED** [review]
Looks good!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**